src/quo2/components/list_items/preview_list/view.cljs

```diff
@@ -0,0 +1,181 @@
+               :border-radius {:rounded 8 :squared 8}
+               :hole-radius   {:rounded 9 :squared 9}
+               :margin-left   -4
+               :hole-size     18
+               :hole-x        11
+               :hole-y        -1}
+   :size/s-14 {:size          14
+               :border-radius {:rounded 7 :squared 7}
+               :hole-radius   {:rounded 8 :squared 8}
+               :margin-left   -2
+               :hole-size     16
+               :hole-x        11
+               :hole-y        -1}})
+
+(defn avatar
+  [item type size-key border-radius]
+  (let [size             (get-in params [size-key :size])
+        user-avatar-size (case size-key
+                           :size/s-32 :small
+                           :size/s-24 :xs
+                           :size/s-20 :xxs
+                           :size/s-16 :xxxs
+                           :xxxs)]
+    (case type
+      :user                                                [user-avatar/user-avatar
+                                                            (merge {:ring?             false
+                                                                    :status-indicator? false
+                                                                    :size              user-avatar-size}
+                                                                   item)]
+
+      :accounts                                            [account-avatar/view
+                                                            (merge item {:size size})]
+
+      (:communities :tokens :collectibles :network :dapps) [fast-image/fast-image
+                                                            {:source (:source item)
+                                                             :style  {:width         size
+                                                                      :height        size
+                                                                      :border-radius border-radius}}]
+      nil)))
+
+(defn list-item
+  [index type size item number margin-left
+   hole-size hole-radius hole-x hole-y border-radius]
+  (let [last-item? (= index (- number 1))]
+    [hole-view/hole-view
+     {:style {:margin-left (if (= index 0) 0 margin-left)}
+      :holes (if last-item?
+               []
+               [{:x            hole-x
+                 :y            hole-y
+                 :width        hole-size
+                 :height       hole-size
+                 :borderRadius hole-radius}])}
+     [avatar item type size border-radius]]))
+
+;; This needs to be cleaned up once the "number tag" component is implemented
+;; https://github.com/status-im/status-mobile/issues/17045
+(defn get-overflow-color
+  [blur? blur-light-color blur-dark-color light-color dark-color theme]
+  (if blur?
+    (colors/theme-colors blur-light-color blur-dark-color theme)
+    (colors/theme-colors light-color dark-color theme)))
+
+(def more-icon-for-sizes #{16 14})
```

Comment:
this `def` name is kind of confusing imo 🙃

Reply:
Yeah. For sizes 14 and 16, In the last overflow item, we should the kebab icon instead of `+(remaining items)`.

The whole overflow part needs to be replaced with the "Number tag" component. It was not implemented yet.
https://github.com/status-im/status-mobile/issues/17045

Once it's added, we can pass the same size key to it. 👍 

https://github.com/status-im/status-mobile/pull/17051/files#diff-7e03eab94812b8180e2654d500554953d15c886a8aec4d2d1b8dc0506f3646f9R90